### PR TITLE
[IMP] form status indicator UX changes

### PR DESCRIPTION
**Pull request by bouvyd**
Rationales:
- having a textual display "Unsaved changes" implied to the user that they
had to save
- having buttons "hiding" behind a hover effect on a span of text was
  very unintuitive
- since we don't want an hover anymore, having text buttons would draw a
  lot of attentions => use icons + tootltip instead

### CODE REVIEW

**Comment by bouvyd** [comment]
@robodoo r+

**Comment by bouvyd** [comment]
@robodoo r-

**Comment by bouvyd** [comment]
@robodoo rebase-ff

File: addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml [hunk 1]
```diff
@@ -3,19 +3,37 @@
...
+                    t-att-disabled="props.isDisabled"
+                    data-hotkey="j"
+                    t-on-click.stop="discard"
+                    data-tooltip="Discard changes"
+                    aria-label="Discard changes"                    >
+                    <i class="fa fa-undo fa-fw" />
+                </button>
+            </div>
             <t t-if="!props.model.root.isVirtual">
                 <t t-if="indicatorMode === 'invalid'">
```
Comment (mcm-odoo) [inline addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml:29]: can be merged

File: addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml [hunk 2]
```diff
@@ -3,19 +3,37 @@
...
             </t>
-            <div class="o_form_status_indicator_buttons d-flex" t-att-class="{ 'o_form_status_indicator_buttons_hidden': !(props.model.root.isVirtual or displayButtons) }">
-                <button type="button" class="o_form_button_save btn btn-secondary" t-att-disabled="props.isDisabled" data-hotkey="s" t-on-click.stop="save">Save</button>
-                <button type="button" class="o_form_button_cancel btn btn-secondary" t-att-disabled="props.isDisabled" data-hotkey="j" t-on-click.stop="discard">Discard</button>
-            </div>
         </div>
     </t>
 
-</templates>
+</templates>
```
Comment (mcm-odoo) [inline addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml:39]: the new line at the end of file is removed

File: addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml [hunk 3]
```diff
@@ -3,19 +3,37 @@
...
+                    <i class="fa fa-cloud-upload fa-fw" />
+                </button>
+                <button
+                    type="button"
+                    class="o_form_button_cancel btn btn-light btn-sm"
+                    t-att-disabled="props.isDisabled"
+                    data-hotkey="j"
+                    t-on-click.stop="discard"
+                    data-tooltip="Discard changes"
+                    aria-label="Discard changes"                    >
```
Comment (pparidans) [inline addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml:24]: that spacing is ________________________long________________________ before the `>` :grin:

File: addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml (hunk 2 above)
Comment (pparidans) [inline addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml:39]: Missing line break

File: addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml (hunk 3 above)
Comment (bouvyd) [reply addons/web/static/src/views/form/form_status_indicator/form_status_indicator.xml:24]: image

**Comment by bouvyd** [comment]
@robodoo r+

**Comment by bouvyd** [comment]
goddammit it won't batch it with its enterprise counterpart
@robodoo r-

**Comment by bouvyd** [comment]
@robodoo r+

**Pull request closed without merging**